src/components/Notifications/DataAlertModal/DataAlertModal.js

```diff
@@ -516,7 +510,6 @@ class DataAlertModal extends React.Component {
           messageInput={this.state.messageInput}
           onTitleInputChange={(e) => this.setState({ titleInput: e.target.value })}
           onMessageInputChange={(e) => this.setState({ messageInput: e.target.value })}
-          queryText={this.getQueryText()}
```

Comment:
We still need to keep this I think. We can display the query if `this.props.queryResponse?.data?.data?.text` exists